packages/core/src/config/types.ts

```diff
@@ -176,3 +177,15 @@ export type RulesFields =
   | 'oas2Decorators'
   | 'oas3_0Decorators'
   | 'oas3_1Decorators';
+
+export enum ConfigArgsOverrideFields {
+  files = 'files',
+  region = 'region',
+}
+
+export const CONFIG_ARGS_OVERRIDE_FIELDS = [
+  ConfigArgsOverrideFields.files,
+  ConfigArgsOverrideFields.region,
+];
+
+export type ArgsType = { [key: string]: any };
```

Comment:
Can we write sth like this: 
```suggestion
export type ArgsType = A1 | A2 | A3...
```
?